docs/management/connectors/action-types/inference.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[inference-action-type]]
+== {inference} connector and action
```

Comment:
```suggestion
== {inference} connector and action
```

` {inference}` doesn't resolve to anything right now. 

You'll either need to write out the full term, or add a variable to this file: https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

Reply:
Fixed by using existing {infer} variable.